patches/CMakeLists.txt

```diff
@@ -7,10 +7,10 @@ find_package(catkin)
 # check arch and os
 execute_process(COMMAND bash -c "gcc -dumpmachine" OUTPUT_VARIABLE gcc_dump_machine OUTPUT_STRIP_TRAILING_WHITESPACE)
 message("-- gcc dumpmachine returns ${gcc_dump_machine}")
-if(gcc_dump_machine MATCHES "x86_64-linux-.*")
+if(gcc_dump_machine MATCHES "x86_64-linux-.*" OR gcc_dump_machine MATCHES "x86_64-redhat-linux")
```

Comment:
It will not. The string is exactly `x86_64-redhat-linux`, no dash after `linux`.